markwiki/views/core.py

```diff
@@ -182,3 +182,10 @@ def search():
     results = app.search_engine.search(user_query)
     return render_template('search.html', user_query=user_query,
                            results=results)
+
+@app.route('/reindex/')
+def reindex():
+    results = app.search_engine.create_index(app.config['WIKI_PATH'])
+    flash('Search index re-created', 'info')
+    '''Display the MarkWiki main page.'''
```

Comment:
I don't think this comment is really helping and it's a docstring in the middle of an implementation block which is a bit odd. Please remove this line.